cniconf/watcher.go

```diff
@@ -125,3 +128,30 @@ func (w *watcher) apply(network metadata.Network) error {
 
 	return lastErr
 }
+
+func UpdateCNIConfigByKeywords(config interface{}, host metadata.Host) interface{} {
+	props, isMap := config.(map[string]interface{})
+	if !isMap {
+		return config
+	}
+
+	for aKey, aValue := range props {
+		v, isString := aValue.(string)
+		if !isString {
+			props[aKey] = UpdateCNIConfigByKeywords(aValue, host)
+		} else {
+			tempSlice := strings.Split(v, ":")
```

Comment:
this should be SplitN(v, ":", 2) so that labels with `:` in it don't cause issues